web/angular-wallet/src/index.html

```diff
@@ -4,7 +4,7 @@
     <head>
 
         <title>BURST</title>
-        <base href="./">
+        <base href="/">
```

Comment:
must be `./` to work in electron. Otherwise we have loading issues of assets....does this would have an impact on the web app?

Reply:
Yeah, without this, the app is broken if refreshed while on a deep link.